test/plots/barPlotTests.ts

```diff
@@ -1213,5 +1213,26 @@ describe("Plots", () => {
       assert.deepEqual(yScale.domain(), [-2.5, 2.5], "domain has been adjusted to visible points");
       svg.remove();
     });
+
+    it("plot auto domain works when there is one bar", () => {
+      let svg = TestMethods.generateSVG();
+
+      let xScale = new Plottable.Scales.Linear();
+      let yScale = new Plottable.Scales.Linear();
+      let xPoint = (xScale.domain()[0] + xScale.domain()[1]) / 2 + 10;
```

Comment:
I wanted to use an `xPoint` that is outside the default domain.  I'll change the code to make this a bit more obvious